common/retry_delays_config.go

```diff
@@ -0,0 +1,73 @@
+package common
+
+import (
+	"fmt"
+
+	commontypes "github.com/agglayer/aggkit/common/types"
+	"github.com/agglayer/aggkit/config/types"
+)
+
+var _ commontypes.RetryPolicyConfigurer = (*RetryDelaysConfig)(nil)
+
+// RetryDelaysConfig defines the configuration for retry delays.
+type RetryDelaysConfig struct {
```

Comment:
WDYT (in order not to be considered that it is unmarshalled from the TOML config)
```suggestion
type RetryDelaysPolicy struct {
```

Reply:
We discussed it, and both are in the fence, that have a part of config and a part of logic, so there are no clear answer so we agree to left as `...Config`